src/libraries/System.IO.FileSystem/tests/RandomAccess/FlushToDisk.cs

```diff
@@ -0,0 +1,103 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO.Pipes;
+using System.Security.Cryptography;
+using Microsoft.Win32.SafeHandles;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public partial class RandomAccess_FlushToDisk : RandomAccess_Base<long>
+    {
+        // Setting this to false disables the ThrowsArgumentOutOfRangeExceptionForNegativeFileOffset() test
+        // which is not applicable to FlushToDisk() since FlushToDisk() does not take a file offset parameter.
```

Comment:
You are correct. I have fixed this in [this commit](https://github.com/dotnet/runtime/pull/89100/commits/a7d6289990c02f84d6d2b79fbd7959ded2ceee74) by removing the comment entirely, along with other non-essential comments in order to avoid issues like that 👍